datafusion/sqllogictest/test_files/order.slt

```diff
@@ -578,3 +578,209 @@ SortPreservingMergeExec: [log_c12_base_c11@0 DESC]
 
 statement ok
 drop table aggregate_test_100;
+
+
+# Sort with lots of repetition values
+# Test sorting a parquet file with 2 million records that has lots of values that are repeated
+statement ok
+CREATE EXTERNAL TABLE repeat_much STORED AS PARQUET LOCATION '../../parquet-testing/data/repeat_much.snappy.parquet';
```

Comment:
I think you could use a path like this and leave the file at its current location:

```suggestion
CREATE EXTERNAL TABLE repeat_much STORED AS PARQUET LOCATION '../../datafusion/core/tests/data/repeat_much.snappy.parquet';
```
If it is only used in sqllogictests, maybe you could move the file to 

```
datafusion/sqllogictest/data/repeat_much.snappy.parquet
```

instead

Reply:
Great - yes it isn't used anywhere else so I moved it to `datafusion/sqllogictest/data/` as suggested by you